src/Animated.test.js

```diff
@@ -0,0 +1,102 @@
+import { getNumberOfInitializedNodes_forTestsOnly as numberOfNodes } from './core/AnimatedNode';
+import Animated, { Easing } from './Animated';
+import AnimatedAlways from './core/AnimatedAlways';
+
+jest.mock('./ReanimatedEventEmitter');
+jest.mock('./ReanimatedModule');
+
+const { Value, timing, spring, decay } = Animated;
+expect.extend({
+  toAttachNodesProperly(animation) {
```

Comment:
This does not seem like a legitimate usecase of `expect.extend` this could be just a method that takes animation object and then evaluates some assertions.